.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: required
+matrix:
+  include:
+    - language: node_js
+      node_js:
+      - "stable"
+      cache:
+        yarn: true
+        directories:
+          - node_modeules
+      install:
+        - yarn
+      scripts:
+        - yarn test
+        - yarn build
+    
+    - language: python
```

Comment:
@parthsujalshah split the pre-commit tests and django's run and compile different!
Refer to this 
https://github.com/developer-student-club-thapar/officialWebsite/blob/master/.travis.yml

Reply:
Also, add comments in the codebase!